ca_description/urdf/create_base.xacro

```diff
@@ -53,7 +54,14 @@
     <xacro:create_wheel prefix="left" y_offset="${wheel_separation / 2}" wheel_radius="${wheel_radius}" wheel_width="${wheel_width}"/>
     <xacro:create_wheel prefix="right" y_offset="${wheel_separation / -2}" wheel_radius="${wheel_radius}" wheel_width="${wheel_width}"/>
     <xacro:caster_wheel/>
-
+    <xacro:color_sensor parent="base_link" prefix="right">
+      <origin xyz="0.18 .08 0.1"  rpy="0 ${0.7*(pi/2)} 1"/>
+      <origin xyz="0.05 0 0" rpy="${-pi/2} 0 ${-pi/2}" />
```

Comment:
This `origin` block is the same for both cases. Is it really necessary? I mean, the position of the optical link w.r.t. the base of the model will be always the same. So, do you need to specify it here?

Reply:
I'm not sure what the point of adding the `*origin` and `**origin` parameters to the macro was, would you like me to remove them again?

Reply:
No, since that parameter is always the same for different color sensor, there is no need to add it here.